ingestion/src/lib.rs

```diff
@@ -130,7 +158,7 @@ pub async fn insert_data(data: Data, conn: &mut PooledPgConn<'_>) -> Result<(),
 }
 
 pub mod kldata;
-use kldata::{filter_and_label_kldata, parse_kldata};
+use kldata::{filter_and_label_kldata, parse_kldata, ObsType};
```

Comment:
I think it would make sense for ObsType to live outside the kldata mod (I guess we will need to create a util mod so kldata can import it), since it's general to ingestion, not specific to kldata/obsinn, and other future sources will need it too.

Reply:
Yeah that's good 👍